### PR TITLE
Issue template: add references what to put in a bug report

### PR DESCRIPTION
**Pull request by fedepaol**
Expanding the troubleshooting guide so it's clearer to users what to try and how to collect the information required in a given issue.

Since this references the new troubleshooting guide material, this must stay on hold until we update the metallb docs, merging https://github.com/metallb/metallb/pull/1958 and doing a release

### CODE REVIEW

**Review by oribon — APPROVED** [review]
lgtm!

**Comment by Ornias1993** [comment]
If you do it like this, it creates GUI fields within github, instead of just pointers:
https://github.com/truecharts/.github/edit/main/.github/ISSUE_TEMPLATE/bug_report.yaml

**Comment by Ornias1993** [comment]
Yeah, all those semi-hidden features :)

File: .github/issue_template.md [hunk 1]
```diff
@@ -1,16 +1,58 @@
...
+
+ - MetalLB Version:
+<!-- [e.g. 0.13.9, 0.12.0] -->
+ - Deployment method:
+<!-- e.g. helm chart, operator, manifests -->
+ - Main CNI:
+<!-- e.g. calico, cilium -->
+ - Kubernetes version:
+<!-- e.g. 1.26.3 -->
+ - Cluster distributin:
```
Comment (liornoy) [inline .github/issue_template.md:16]: nit: distribution

**Pull request closed without merging**